applications/add_file_to_db.py

```diff
@@ -61,10 +60,10 @@ def _userConfirm():
 
 def main():
 
-    db = db_handler.DatabaseHandler()
-
-    parser = argparser.CommandLineParser(description=("Add a file or files to the DB."))
-    group = parser.add_mutually_exclusive_group(required=True)
+    # temporary solution; class values in db_handler
+    _tmp_db_config = db_handler.DatabaseHandler()
+    config = configurator.Configurator(description=("Add a file or files to the DB."))
+    group = config.parser.add_mutually_exclusive_group(required=True)
     group.add_argument(
         "-f",
         "--fileName",
```

Comment:
Is `fileName` a required command line argument?

Reply:
Yes, it is required